src/editor/deserialize.ts

```diff
@@ -195,6 +195,8 @@ function parseNode(n: Node, pc: PartCreator, opts: IParseOptions, mkListItem?: (
                     return [pc.plain("**"), ...parseChildren(n, pc, opts), pc.plain("**")];
                 case "DEL":
                     return [pc.plain("<del>"), ...parseChildren(n, pc, opts), pc.plain("</del>")];
+                case "S":
+                    return [pc.plain("<s>"), ...parseChildren(n, pc, opts), pc.plain("</s>")];
```

Comment:
this is greek to me. Presumably something to do with one of our rich-text-editor implementations?

Reply:
For our normal composer, when parsing html for editing a message